clang/include/clang/Basic/DiagnosticSemaKinds.td

```diff
@@ -11887,6 +11887,9 @@ def err_sycl_add_ir_attribute_invalid_value : Error<
 def err_sycl_add_ir_attribute_invalid_filter : Error<
   "initializer list in the first argument of %0 must contain only string "
   "literals">;
+def warn_sycl_old_and_new_kernel_attributes : Warning<
+  "kernel has attribute %0 but also has kernel properties. Conflicting "
```

Comment:
Sorry for bringing this up so late, but I don't like this diagnostic. I prefer something like - 

`kernel has attribute %0 and kernel properties. Conflicting properties will be ignored`

Feel free to ignore suggestion if you/other reviewers prefer current diagnostic.

Reply:
I prefer the current version of the spelling.   But I don't feel strongly about it either.  :-)

Reply:
Works for me!